internal/generate/tagstests/main.go

```diff
@@ -22,7 +22,7 @@ import (
 	"text/template"
 
 	"github.com/YakDriver/regexache"
-	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+	acctestconsts "github.com/hashicorp/terraform-provider-aws/internal/acctest/const"
```

Comment:
I've gone back and forth on the location of this stuff (in my mind). My thoughts are maybe names is the right place since it's lightweight. Most of the time (~99%), where the constants are used, `acctest` is already imported. The one exception being generators. Or maybe we have a whole new package with attribute and testing consts that just dedicated to constants. Depends on if you're splitting functionality horizontally (constant stuff) or vertically (test/non-test), as it were.

Reply:
I like the testing constants being in the `acctest` package, since it scopes them to tests. In this case, the only thing in `acctest/const` is the function `ConstOrQuote`, so maybe the name `const` is wrong for the package.

The generator never uses the contstants in `acctest`, it only needs to map from string values to the constant names. Essentially, `ConstOrQuote` is a "generator helper function", so it isn't directly part of `acctest` but is strongly-associated with it